hw1/02-palindrome.js

```diff
@@ -1,3 +1,29 @@
-const elem = document.querySelector('input');
+const input = document.querySelector('input');
+const result = document.getElementById('result');
 
-elem.addEventListener('input', handleInput);
+input.addEventListener('input', handleInput);
+
+function handleInput(e) {
+  const inputNumberStr = e.target.value;
+  if (inputNumberStr.length < 1) {
+    result.textContent = '';
+  } else if (isBadInput(inputNumberStr)) {
+    result.style.color = 'red';
+    result.textContent = 'Invalid Input Error: Only positive numbers allowed.';
+  } else if (isPalindrome(inputNumberStr)) {
+    result.style.color = 'green';
+    result.textContent = 'Yes. This is a palindrome!';
+  } else {
+    result.style.color = 'red';
```

Comment:
_Accessibility_ - One _very small_ accessibility issue detected with WAVE is that the text color changes using `result.style.color` create contrast errors with the background.
Although the colors you used appear almost exactly the same as the screenshots we were given and you even used a white background, it turns out CSS named colors are just _slightly_ different from the default red and green colors used by Bootstrap (`.text-success` and `.text-danger`).  
I apologize if this seems extremely nitpicky!  
If you do want to change it, one way could be to modify `result`'s classes using either `.text-success` or `.text-danger` as necessary.

Reply:
I ran wave on my page, but forgot to run it on the text color! Thanks for the catch here. The fix is in the updated PR.